tools/docker/Dockerfile.webkitgtk

```diff
@@ -29,19 +33,11 @@ RUN apt-get -qqy update \
     xvfb \
     git-core
 
-# Configure buster-wpt-webkit-updates repository
-RUN apt-key adv --fetch-keys https://webkitgtk.org/debian/apt.key
-RUN printf 'deb [arch=amd64] https://webkitgtk.org/debian buster-wpt-webkit-updates main' \
-    > /etc/apt/sources.list.d/webkitgtk.list
-RUN printf 'Package: *\nPin: origin webkitgtk.org\nPin-Priority: 1999' \
-    > /etc/apt/preferences.d/99webkitgtk
-
-# Update and upgrade
-RUN apt-get update \
-  && apt-get -y upgrade
 
-# Install webkit packages from https://webkitgtk.org/debian
-RUN apt-get install -y -t buster-wpt-webkit-updates webkit2gtk-driver
+# To speed up testers, install in the image most of WebKitGTK dependencies
```

Comment:
Should this say "to speed up setup" or similar? But more importantly, does adding some of the dependencies not mean that an `apt-get upgrade` will be necessary as well in run_tc.py? Or does `apt-get install` implicitly upgrade dependencies of the thing being installed?

Reply:
Yes, ```apt-get install``` implicitly upgrades the needed dependencies. But running an extra ```apt-get upgrade``` is maybe a good idea to upgrade unrelated dependencies and ensure it always runs with the last versions of the packages available.

Reply:
Pushed a new commit implementing this